docs/DeveloperGuide.md

```diff
@@ -1354,15 +1357,21 @@ otherwise)
 
 **Extensions:**
 
-- 1a. Logged-in Owner specifies invalid customer.
+- 1a. Logged-in Owner specifies a customer ID that does not exist is a positive integer.
```

Comment:
Would this be considered too much internal details. Also I think the phrasing is a bit weird.